### PR TITLE
Issues/6 add to metamask button

### PR DESCRIPTION
**Pull request by dev0614**

### CODE REVIEW

File: apps/dex/.env.testnet
Comment (fredev) [inline apps/dex/.env.testnet:1]: revert the changes to this file

File: apps/dex/package.json [hunk 1]
```diff
@@ -67,6 +67,7 @@
     ]
   },
   "devDependencies": {
-    "@types/big.js": "^6.2.2"
+    "@types/big.js": "^6.2.2",
+    "dotenv-cli": "^7.4.2"
```
Comment (fredev) [inline apps/dex/package.json:71]: dotenv-cli is not needed, remove the dependency

File: apps/dex/src/components/AddTokenToMetamaskButton.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Button, Tooltip } from '@mui/material';
+import metamask from '../img/metamask.png';
+
+interface AddToMetaMaskButtonProps {
+  token:
```
Comment (fredev) [inline apps/dex/src/components/AddTokenToMetamaskButton.tsx:6]: can we use `Currency` for this type?

File: apps/dex/src/components/Web3Provider.tsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ import { ReactNode } from 'react';
 const queryClient = new QueryClient();
 
 const projectId = process.env.REACT_APP_WALLECT_CONNECT_PROJECT_ID ?? '';
-
+console.log(projectId)
```
Comment (fredev) [inline apps/dex/src/components/Web3Provider.tsx:29]: remove console.log

**Review by fredev — CHANGES_REQUESTED** [review]

File: apps/dex/src/components/Web3Provider.tsx (hunk 3 above)
Comment (dev0614) [reply apps/dex/src/components/Web3Provider.tsx:29]: Will delete the console.

File: apps/dex/src/components/AddTokenToMetamaskButton.tsx (hunk 2 above)
Comment (dev0614) [reply apps/dex/src/components/AddTokenToMetamaskButton.tsx]: Will fix that

File: apps/dex/package.json (hunk 1 above)
Comment (dev0614) [reply apps/dex/package.json:71]: Without dotenv-cli, couldn't import env variables.
If it's not necessary, you can delete it.

**Review by fredev — APPROVED** [review]

**Pull request merged**